script_files/main.js

```diff
@@ -1,27 +1,38 @@
-import {displayStats} from "./displayStats.js";
+import {displayStats, displayTotalStats} from "./displayStats.js";
 import {displayTodayStats} from "./displayStats.js";
 import {generateText} from "./generateText.js";
 import {markCurrentChar} from "./displayText.js";
 import {handleKeyDownEvent} from "./handleKeyDownEvent.js";
 import {incrementSeconds} from "./timeHandler.js";
+import {handleLocalStorage} from "./localStorageHandler.js";
+import {resetLocalStorageForTodayStats} from "./localStorageHandler.js";
+import {resetLocalStorageForTotalStats} from "./localStorageHandler.js";
 
 const generateTextButton = document.getElementById('generate-text-button');
 const resetProgressButton = document.getElementById('reset-button');
 const statsText = document.getElementById('last-set-stats-text');
 const statsTextForSeconds = document.getElementById('last-set-stats-time-text');
 const todayStatsText = document.getElementById('today-stats-text');
+const totalStatsText = document.getElementById('total-stats-text');
```

Comment:
```suggestion
const totalStatsTextContainer = document.getElementById('total-stats-text');
```

This change would clarify that this variable is referencing an HTML element, rather than a text/string :) You could change this for other occurrences as well.